flowmachine/flowmachine/features/subscriber/proportion_outgoing.py

```diff
@@ -16,7 +16,7 @@
 from .metaclasses import SubscriberFeature
 
 
-class ProportionOutgoing(SubscriberFeature):
+class ProportionEventOutgoing(SubscriberFeature):
```

Comment:
Why's this been renamed? And if it needs to be renamed,  please rename file to correspond.

Reply:
I have actually deleted this class in PR#348 as it becomes redundant with the introduction of `ProportionEventType`. If you think we will want to keep this class I can make the requested changes.

In any case, I renamed this class to align with `ProportionEventReciprocal` which was introduced here.

Reply:
I revert it back for now since we'll delete in another PR.